### PR TITLE
some typos

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] Was this **discussed/approved** via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the **[contributor guideline](https://github.com/PyTorchLightning/lightning-flash/tree/master/.github/CONTRIBUTING.md)**, Pull Request section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes?
- [ ] Did you write any **new necessary tests**? [not needed for typos/docs]
- [ ] Did you verify **new and existing tests pass** locally with your changes?
- [ ] If you made a notable change (that affects users), did you **update the [CHANGELOG](https://github.com/PyTorchLightning/lightning-flash/blob/master/CHANGELOG.md)**?

## PR review
 - [x] Is this pull request **ready for review**? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: flash/image/instance_segmentation/cli.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def from_pets(
     batch_size: int = 1,
     **data_module_kwargs,
 ) -> InstanceSegmentationData:
-    """Downloads and loads the pets data set from icedata."""
+    """Downloads and loads the pets' data set from icedata."""
```
Comment (krshrimali) [inline flash/image/instance_segmentation/cli.py:42]: I think it should be `pet` here. (https://airctic.github.io/icedata/pets/)

```suggestion
    """Downloads and loads the pet data set from icedata."""
```

File: flash/text/question_answering/collate.py [hunk 2]
```diff
@@ -96,7 +96,7 @@ def _prepare_val_features(self, samples: Any, tokenized_samples: Any, pad_on_rig
         sample_mapping = tokenized_samples.pop("overflow_to_sample_mapping")
 
         # For evaluation, we will need to convert our predictions to substrings of the context, so we keep the
-        # corresponding example_id and we will store the offset mappings.
+        # corresponding example_id, and we will store the offset mappings.
```
Comment (krshrimali) [inline flash/text/question_answering/collate.py:99]: I think in this PR, we made a change like: `parameters' dictionary'. If that's right, probably it should be `offset's mappings` here (if we mead `mappings of the offset` here and not `offset mappings` term)? Not a native English speaker, so not too sure.

**Review by krshrimali — CHANGES_REQUESTED** [review]
Thanks, @Borda - great work on catching typos :) This is useful. I have a few comments on some of them, not a native English speaker, so I may be wrong.

File: flash/core/adapter.py [hunk 3]
```diff
@@ -67,7 +67,7 @@ def identity_collate_fn(x):
 
 class AdapterTask(Task):
     """The ``AdapterTask`` is a :class:`~flash.core.model.Task` which wraps an :class:`~flash.core.adapter.Adapter`
-    and forwards all of the hooks.
+    and forwards all the hooks.
```
Comment (ethanwharris) [inline flash/core/adapter.py:70]: I think it's better witht the `of`:
```suggestion
    and forwards all of the hooks.
```

File: flash/core/integrations/fiftyone/utils.py [hunk 4]
```diff
@@ -29,7 +29,7 @@ def visualize(
     """Visualizes predictions from a model with a FiftyOne Output in the
     :ref:`FiftyOne App <fiftyone:fiftyone-app>`.
 
-    This method can be used in all of the following environments:
+    This method can be used in all the following environments:
```
Comment (ethanwharris) [inline flash/core/integrations/fiftyone/utils.py:32]:
```suggestion
    This method can be used in all of the following environments:
```

File: flash/core/serve/dag/utils.py [hunk 5]
```diff
@@ -26,7 +26,7 @@ def funcname(func):
     if "numpy" in module_name and type_name == "vectorize":
         return ("vectorize_" + funcname(func.pyfunc))[:50]
 
-    # All other callables
+    # All others callables
```
Comment (ethanwharris) [inline flash/core/serve/dag/utils.py:29]:
```suggestion
    # All other callables
```

File: flash/core/trainer.py [hunk 6]
```diff
@@ -170,7 +170,7 @@ def predict(
         r"""
         Run inference on your data.
         This will call the model forward function to compute predictions. Useful to perform distributed
-        and batched predictions. Logging is disabled in the predict hooks.
+        and batched predictions. Logging is disabled in the predictiom hooks.
```
Comment (ethanwharris) [inline flash/core/trainer.py:173]:
```suggestion
        and batched predictions. Logging is disabled in the prediction hooks.
```

File: flash/tabular/forecasting/data.py [hunk 7]
```diff
@@ -42,7 +42,7 @@ class TabularForecastingData(DataModule):
 
     @property
     def parameters(self) -> Optional[Dict[str, Any]]:
-        """The parameters dictionary from the ``TimeSeriesDataSet`` object created from the train data when
+        """The parameters' dictionary from the ``TimeSeriesDataSet`` object created from the train data when
```
Comment (ethanwharris) [inline flash/tabular/forecasting/data.py:45]:
```suggestion
        """The ``parameters`` dictionary from the ``TimeSeriesDataSet`` object created from the train data when
```

**Review by ethanwharris — APPROVED** [review]

File: flash/core/serve/core.py [hunk 8]
```diff
@@ -183,7 +183,7 @@ def __str__(self):
 
 @dataclass
 class Parameter:
-    """Holder class for each grid type of a component and connections from those to the types of other components.
```
[the remark this answers is not in the document]
Comment (Borda) [reply flash/core/serve/core.py:186]: mind send suggestion :)

File: flash/text/question_answering/collate.py (hunk 2 above)
Comment (Borda) [reply flash/text/question_answering/collate.py:99]: mind send suggestion :)

**Review by krshrimali — APPROVED** [review]
Added suggestions, if they look good, please commit and this should be good to go 🚀 Thanks @Borda!

**Comment by Borda** [comment]
@krshrimali seems you merged it before I had a chance to accept them...

**Pull request merged**